default-configs/xtensa-softmmu.mak

```diff
@@ -9,5 +9,7 @@ CONFIG_XTENSA_VIRT=y
 CONFIG_XTENSA_XTFPGA=y
 CONFIG_SSP=y
 CONFIG_DW_DMA_DSP=y
+CONFIG_EDMA_DSP=y
```

Comment:
@dianacretu Would be easier to review and later refer this commits if we can split it in two. 1 commit for EDMA and 1 commit for SDMA.